presto-hive/src/main/java/com/facebook/presto/hive/HiveSessionProperties.java

```diff
@@ -215,6 +216,11 @@ public HiveSessionProperties(HiveClientConfig hiveClientConfig, OrcFileWriterCon
                         "Experimental: ORC: Compression level, works only for ZSTD and ZLIB compression kinds",
                         orcFileWriterConfig.getCompressionLevel(),
                         false),
+                booleanProperty(
+                        ORC_USE_VECTOR_FILTER,
+                        "Experimental: enable vector path for orc filters",
```

Comment:
same comment about noting the jdk requirement and probably removing "experimental". Would be great to add validation that the jdk meets the minimum version if you set this, and fail with INVALID_SESSION_PROPERTY otherwise.